GoogleChromeLabs/ps-analysis-tool
# Feature: Add insights card to RWS panel

## Description
This PR includes code to add insights card inside the RWS panel. The insights card includes basic info showing whether a page is part of Related Website Sets alongside their primary domain and rationale.


## Relevant Technical Choices
- Using cli script to store RWS JSON file inside the extension.




## Testing Instructions
- Open `https://hindustantimes.com` or `https://livemint.com`, visit `Site Boundaries` -> `Related Website Sets`.
- On the landing page, you'll see the text telling site being a part of RWS.
- Open the RWS JSON file on the RWS repo and compare the presence of data inside that.



## Additional Information:



## Screenshot/Screencast

<img width="693" alt="Screenshot 2023-09-15 at 4 27 19 PM" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/58820001/a7eeeb56-570b-45cf-81f3-1f8d93f1ad80">

<img width="681" alt="Screenshot 2023-09-15 at 4 27 40 PM" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/58820001/8bd77737-a659-46c7-91b0-1906f545915e">



---



Partially addresses #120
